Without this home directories hosted on NFS cause performance issues.
This should be investigated if this is still needed.

See: https://bugzilla.mozilla.org/show_bug.cgi?id=1428169

--- thunderbird-140.1.0/comm/mail/app/profile/all-thunderbird.js
+++ thunderbird-140.1.0/comm/mail/app/profile/all-thunderbird.js
@@ -355,6 +355,9 @@ pref("accessibility.typeaheadfind.timeou
 pref("accessibility.typeaheadfind.linksonly", false);
 pref("accessibility.typeaheadfind.flashBar", 1);
 
+// Whether profile is on NFS
+pref("storage.nfs_filesystem", true);
+
 pref("mail.close_message_window.on_delete", false);
 
 // Number of lines of To/CC/BCC address headers to show before "more"
